src/java/org/apache/cassandra/locator/LocalStrategy.java

```diff
@@ -70,7 +71,7 @@ public void validateOptions() throws ConfigurationException
     }
 
     @Override
-    public void maybeWarnOnOptions()
+    public void maybeWarnOnOptions(ClientState state)
```

Comment:
please revert, you do not have a body so you should take in the default implementations